macros/parsers/parserCheckboxList.pl

```diff
@@ -292,14 +300,24 @@ sub addLabels {
 
 	my $choices = $self->{orderedChoices};
 	my $labels  = $self->{labels};
-	$labels = [ 1 .. $self->{n} ]                        if $labels eq '123';
-	$labels = [ @main::ALPHABET[ 0 .. $self->{n} - 1 ] ] if uc($labels) eq 'ABC';
-	$labels = []                                         if $labels eq 'text';
+	my @roman =
+		qw(i ii iii iv v vi vii viii ix x xi xii xiii xiv xv xvi xvii xviii xix xx xxi xxii xxiii xxiv xxv xxvi);
+	$labels = [ 1 .. $self->{n} ] if $labels eq '123';
+	$labels = [ map { lc($_) } @main::ALPHABET[ 0 .. $self->{n} - 1 ] ]
+		if $labels eq 'abc';
+	$labels = [ @main::ALPHABET[ 0 .. $self->{n} - 1 ] ]
+		if uc($labels) eq 'ABC' && $labels ne 'abc';
+	$labels = [ @roman[ 0 .. $self->{n} - 1 ] ] if $labels eq 'roman';
+	$labels = [ map { uc($_) } @roman[ 0 .. $self->{n} - 1 ] ]
+		if uc($labels) eq 'ROMAN' && $labels ne 'roman';
+	$labels = [] if $labels eq 'text';
 
 	if (ref($labels) ne 'ARRAY') {
 		my $replace = $labels ne 'auto';
 		if (!$replace) {
-			for (@$choices) { $replace = 1 if $_ =~ m/[^-+.,;:()!\[\]a-z0-9 ]/i }
+			for (@$choices) {
+				$replace = 1 if $_ =~ m/[^-+.,;:()!\[\]a-z0-9 ]/i;
+			}
```

Comment:
I don't like this change.  In fact some perl style guides say the way it was is preferred for one line blocks.  Note that if on one line the semicolon should be omitted.  See https://perldoc.perl.org/perlstyle.